testcases/MachineConfig.py

```diff
@@ -79,6 +79,15 @@ def runTest(self):
                 status = CecConfig(self.cv_HMC,self.system_name,self.lpar_name,self.lpar_prof,lmb_size).CecSetup_lmb()
                 if status:
                     self.fail(status)
+
+            if "hugepages" in config_value:
```

Comment:
what can be ideal way is cec in config_value and call a common function which helps to do power off and power on of system in single call